clients/linux/src/app/imp_settings_dialog.rs

```diff
@@ -400,7 +400,7 @@ fn payment_err_to_string(err: lb::Error<lb::SwitchAccountTierError>) -> String {
             CurrentUsageIsMoreThanNewTier => {
                 "Your current usage is greater than the data cap of your desired subscription tier."
             }
-            ConcurrentRequestsAreTooSoon => "ConcurrentRequestsAreTooSoon",
+            TooManyRequestsTooSoon => "ConcurrentRequestsAreTooSoon",
```

Comment:
```suggestion
            TooManyRequestsTooSoon => "Too many requests. Please wait a little while before trying again.",
```
Is this accurate? I wasn't sure what the error was meant to be before lol

Reply:
That is an accurate description of the error.